[CI] MixedClusterClientYamlTestSuiteIT test {p0=bulk/60_deprecated/Deprecated parameters should produce warning in Bulk query}

Edit: This started to fail again since 6.8.8 was created and seems to related #53923. Probably need to disable it for 6.8.8. Asking @henningandersen to take a look.

Branch 6.8: The failure can be reproduced locally with
```
./gradlew ':qa:mixed-cluster:v6.8.8#mixedClusterTestRunner' \
  -Dtests.seed=D0EAA82870892007 \
  -Dtests.class=org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT \
  -Dtests.method="test {p0=bulk/60_deprecated/Deprecated parameters should produce warning in Bulk query}" \
  -Dtests.security.manager=true \
  -Dtests.locale=de-GR \
  -Dtests.timezone=America/Denver \
  -Dtests.distribution=default \
  -Dcompiler.java=12 \
  -Druntime.java=8
```

Build Job:
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.8+default-distro+bwc/BWC_VERSION=6.8.8,nodes=centos-7&&immutable/714/console

Failure email:
https://groups.google.com/a/elastic.co/forum/?utm_medium=email&utm_source=footer#!msg/build-elasticsearch/iZs1dswONuQ/Zj74fQrbAwAJ

Build scan:
https://gradle-enterprise.elastic.co/s/k35eh57sjole6

Somehow it is difficult to locate the actual error message in the build scan. It is pasted here:

```
java.lang.AssertionError: Failure at [bulk/60_deprecated:10]: got unexpected warning headers [
	299 Elasticsearch-6.8.8-2f4c224 "Usage of internal versioning for optimistic concurrency control is deprecated and will be removed. Please use the `if_seq_no` and `if_primary_term` parameters instead. (request for index [test_index], type [test_type], id [test_id_1])"
	299 Elasticsearch-6.8.8-2f4c224 "Usage of internal versioning for optimistic concurrency control is deprecated and will be removed. Please use the `if_seq_no` and `if_primary_term` parameters instead. (request for index [test_index], type [test_type], id [test_id_2])"
]
did not get expected warning header [
	Usage of internal versioning for optimistic concurrency control is deprecated and will be removed. Please use the `if_seq_no` and `if_primary_term` parameters instead. (request for index [test_index], type [test_type])
]

```

--- a/rest-api-spec/src/main/resources/rest-api-spec/test/bulk/60_deprecated.yml
+++ b/rest-api-spec/src/main/resources/rest-api-spec/test/bulk/60_deprecated.yml
@@ -3,8 +3,8 @@
 "Deprecated parameters should produce warning in Bulk query":
 
    - skip:
-       version: " - 6.8.7"
-       reason:  versioned operations were deprecated in 6.7 but we won't issue deprecation unless all nodes are upgraded. 6.8.0-6.8.7 skipped due to new message
+       version: " - 6.8.8"
+       reason:  versioned operations were deprecated in 6.7 but we won't issue deprecation unless all nodes are upgraded. 6.8.0-6.8.8 skipped due to new message
        features: "warnings"
 
    - do:

--- a/rest-api-spec/src/main/resources/rest-api-spec/test/delete/20_internal_version.yml
+++ b/rest-api-spec/src/main/resources/rest-api-spec/test/delete/20_internal_version.yml
@@ -1,8 +1,8 @@
 ---
 "Internal version":
  - skip:
-      version: " - 6.8.7"
-      reason:  versioned operations were deprecated in 6.7 but deprecation warnings are not consistent until all nodes on 6.8.8+
+      version: " - 6.8.8"
+      reason:  versioned operations were deprecated in 6.7 but deprecation warnings are not consistent until all nodes on 6.8.9+
       features: warnings
 
  - do:

--- a/rest-api-spec/src/main/resources/rest-api-spec/test/index/30_internal_version.yml
+++ b/rest-api-spec/src/main/resources/rest-api-spec/test/index/30_internal_version.yml
@@ -1,8 +1,8 @@
 ---
 "Internal version":
  - skip:
-     version: " - 6.8.7"
-     reason:  versioned operations were deprecated in 6.7 but deprecation warnings are not consistent until all nodes on 6.8.8+
+     version: " - 6.8.8"
+     reason:  versioned operations were deprecated in 6.7 but deprecation warnings are not consistent until all nodes on 6.8.9+
      features: warnings
 
  - do:

--- a/rest-api-spec/src/main/resources/rest-api-spec/test/update/30_internal_version.yml
+++ b/rest-api-spec/src/main/resources/rest-api-spec/test/update/30_internal_version.yml
@@ -1,8 +1,8 @@
 ---
 "Internal version":
  - skip:
-      version: " - 6.8.0"
-      reason:  versioned operations were deprecated in 6.7 but deprecation warnings are not consistent until all nodes on 6.8.1+
+      version: " - 6.8.8"
+      reason:  versioned operations were deprecated in 6.7 but deprecation warnings are not consistent until all nodes on 6.8.9+
       features: warnings
 
  - do:

